tests/dummy/app/pods/docs/quickstart/template.md

```diff
@@ -63,28 +100,30 @@ This quickstart guide will get you going with a docs site for your brand new add
     </div>
   {{/docs-snippet}}
 
-8. **Add more docs pages.** It's up to you how to structure your docs - use the Snippet, Viewer and Demo components to help you write your documentation. Let's add another page to ours: we'll add the route, link to our docs viewer, and the actual template.
+7. **Launch your docs site**. Run `ember serve` and browse to
+`localhost:4200/docs` to enjoy your brand new documentation website.
 
-  {{#docs-snippet name='quickstart-more-docs.js' title='tests/dummy/app/router.js'}}
-    this.route('docs', function() {
-      this.route('installation');
-    });
-  {{/docs-snippet}}
+8. **Create more pages.** To add more doc pages simply follow the same steps as
+used for the `Usage` page in above instructions:
```

Comment:
probably should be `page in the above instructions`

Reply:
@pzuraq How about `Usage page in above examples` ?

This way users can reverse engineer/lookup what's needed to create an extra docs page.